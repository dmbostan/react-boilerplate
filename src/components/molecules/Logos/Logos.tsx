import Link from "@atoms/Link/Link.tsx";
import Image from "@atoms/Image/Image.tsx";
import reactLogo from "@assets/react.svg";
import viteLogo from '@assets/vite.svg'

const Logos = () => {
    return(
        <>
            <Link href="https://vitejs.dev" target="_blank">
                <Image src={viteLogo} className="logo" alt="Vite logo" />
            </Link>
            <Link href="https://react.dev" target="_blank">
                <Image src={reactLogo}  className="logo react" alt="React logo" />
            </Link>
        </>
    );
}

export default Logos;
