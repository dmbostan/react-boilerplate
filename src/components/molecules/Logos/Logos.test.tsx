import Logos from "./Logos.tsx";
import {render} from "@testing-library/react";

describe('Logos component', () => {
    it('matches snapshot', () => {
        const { asFragment } = render(<Logos />);

        expect(asFragment()).toMatchSnapshot();
    });
});
