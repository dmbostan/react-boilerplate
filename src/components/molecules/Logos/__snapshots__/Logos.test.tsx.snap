// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`Logos component > should match the snapshot 1`] = `
<DocumentFragment>
  <a
    href="https://vitejs.dev"
    target="_blank"
  >
    <img
      alt="Vite logo"
      class="logo"
      src="/react-boilerplate/src/assets/vite.svg"
    />
  </a>
  <a
    href="https://react.dev"
    target="_blank"
  >
    <img
      alt="React logo"
      class="logo react"
      src="/react-boilerplate/src/assets/react.svg"
    />
  </a>
</DocumentFragment>
`;
